pull updates from base repository

rumititim opened the pull request.

zahorniak committed:
feat: Add new parameter monitoring_role_description (#350)

antonbabenko committed:
Updated CHANGELOG

bryantbiggs committed:
fix: update CI/CD process to use composite actions and auto-release workflow (#361)

semantic-release-bot committed:
chore(release): version 3.4.1 [skip ci]

## [3.4.1](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v3.4.0...v3.4.1) (2021-11-07)

### Bug Fixes

* update CI/CD process to use composite actions and auto-release workflow ([#361](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/361)) ([123ea2a](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/123ea2a136c913de0588d4820b291eb4d1d74d61))

bryantbiggs committed:
fix: update CI/CD process to align auto-release workflow (#366)

bryantbiggs committed:
chore: Update release configuration files to correctly use conventional-commits (#370)

semantic-release-bot committed:
chore(release): version 3.4.2 [skip ci]

### [3.4.2](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v3.4.1...v3.4.2) (2022-01-10)

### Bug Fixes

* update CI/CD process to align auto-release workflow ([#366](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/366)) ([e38aace](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/e38aaced17278567973dad4980fae8757d6a1265))

kwojcik-blockether committed:
feat: Added `latest_restorable_time` to ignore_changes (#356)

Co-authored-by: Anton Babenko <anton@antonbabenko.com>

semantic-release-bot committed:
chore(release): version 3.5.0 [skip ci]

## [3.5.0](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v3.4.2...v3.5.0) (2022-01-12)

### Features

* Added `latest_restorable_time` to ignore_changes ([#356](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/356)) ([77902c2](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/77902c21414f24026d720e72068e589b3855fab8))

bryantbiggs committed:
feat!: Update default values for security posture improvement (#369)

BREAKING CHANGES:

- Support for Terraform `<=v0.12.x` has been dropped; `v0.13.1` is now the minimum supported version
- Terraform AWS provider minimum version is now `v4.0.0` in order to support the replacement of `var.name`(deprecated) for `var.db_name`
- Separate RDS instance resource for MSSQL/SQLServer has been removed - all engines are supported under one resource
- `storage_encrypted` is now set to `true` by default; was previously `false`
- `create_random_password` is now set to `true` by default; was previously `false`
- `create_db_subnet_group` is now set to `false` by default; was previously `true`; typically a shared DB subnet group will be used, most likely from the VPC module
- `random_password_length` is now set to `16` by default, was previously `10`
- Random provider minimum version supported is now `v3.1.0`
- `final_snapshot_identifier` no longer coalesces `var.final_snapshot_identifier` and instead relies on `var.final_snapshot_identifier_prefix` with a random suffix to avoid name collisions

semantic-release-bot committed:
chore(release): version 4.0.0 [skip ci]

## [4.0.0](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v3.5.0...v4.0.0) (2022-02-16)

### ⚠ BREAKING CHANGES

* Update default values for security posture improvement (#369)

### Features

* Update default values for security posture improvement ([#369](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/369)) ([9149ec1](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/9149ec147327fbe5a7675ac48e9e641b5d7ec6e1))

mpiala reviewed (approved):
update from public module, ok

bryantbiggs committed:
feat: Add support for managing CloudWatch log groups (#380)

semantic-release-bot committed:
chore(release): version 4.1.0 [skip ci]

## [4.1.0](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v4.0.0...v4.1.0) (2022-02-17)

### Features

* Add support for managing CloudWatch log groups ([#380](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/380)) ([761fb7f](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/761fb7f5b8d7bb8ae99c54d8fa143c40d0772c0a))

bryantbiggs committed:
fix: Remove check for `is_mssql` which is no longer valid (#382)

semantic-release-bot committed:
chore(release): version 4.1.1 [skip ci]

### [4.1.1](https://github.com/terraform-aws-modules/terraform-aws-rds/compare/v4.1.0...v4.1.1) (2022-02-17)

### Bug Fixes

* Remove check for `is_mssql` which is no longer valid ([#382](https://github.com/terraform-aws-modules/terraform-aws-rds/issues/382)) ([f4481e6](https://github.com/terraform-aws-modules/terraform-aws-rds/commit/f4481e6f4c21452840627b8cd229753229c04e5c))
